src/middlewared/middlewared/utils/osc/freebsd/user_context.py

```diff
@@ -86,6 +86,13 @@ def run_command_with_user_context(commandline, user, callback):
             callback(get)
         except queue.Empty:
             pass
+        except Exception as e:
+            logger.error(
+                'Error encountered: %r', e
+            )
+            p.kill()
+            return
```

Comment:
We probably want to re-raise the error here instead of returning as nothing wrong happened.